src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -59,12 +59,19 @@ public String getPreamble() {
     }
 
     /**
-     * Validates whether multiple prefixes are stored in Argument Multimap.
+     * Validates whether only a single prefix and a single keyword are stored in Argument MultiMap.
      *
-     * @return true if 2 or more prefixes are stored
+     * @return true if there are 2 or more prefix or keywords are stored.
```

Comment:
Maybe this description can be `return true if there are 2 or more prefixes or keywords stored.`?